src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java

```diff
@@ -64,13 +64,13 @@ public void testTimestamp() throws InvalidParametersException, EntityAlreadyExis
         ______TS("success : keep lastUpdated");
```

Comment:
You should remove/or change the description of the test case.

Reply:
I mean either you should remove the test case or change the description of the test case :P

Reply:
I removed the test case as it is not needed and it is already checked.